src/chrome/content/rules/Polygon.com.xml

```diff
@@ -29,6 +29,7 @@
 		<!--	Redirect to http:
 						-->
 		<exclusion pattern="^http://www\.polygon\.com/(?:$|\?)" />
+		<exclusion pattern="^http://www\.polygon\.com/services/entry_stats" />
```

Comment:
@antoniojtorres A `<test url="http://www.polygon.com/services/entry_stats" />` is required here to pass the tests.